api/client/volume/create.go

```diff
@@ -26,10 +26,17 @@ func newCreateCommand(dockerCli *client.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create [VOLUME_NAME]",
 		Short: "Create a volume",
-		Args:  cli.NoArgs,
+		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
+				if opts.name != "" {
+					fmt.Fprint(dockerCli.Err(), "volume create exception either specify --name or provide positional arg, not both")
```

Comment:
Perhaps the text should start with `Conflicting options:` for consistency (e.g. See https://github.com/docker/docker/blob/506234e3f0f6d8f8a0629e09f38dfe96ada025c2/api/client/container/run.go#L88-90)